templates/email/accept.txt

```diff
@@ -4,69 +4,50 @@ Congratulations! Your talk,
-me know.
-
-We'll be announcing the list of accepted talks shortly. If
-you can no longer give this talk, please contact me right away so I
-can find a replacement. If you need to make changes to your title or
-abstract, please contact me.
-
-We haven't yet completed the schedule, so we don't yet know whether
-you've been allocated a 30 or a 45 minute time slot. We'll be
-announcing the schedule shortly, so please stay tuned.
-
-If you have particular scheduling requirements (if you plan to come to
-PyCon late or leave early, for example), please let me know today or
-tomorrow, so that I can take your requirements into account.
-
-We'd like to have a collection of past proposals we can use in the
-future to help potential PyCon speakers out. If you are amenable
-to us publicly sharing your proposal, please reply to this email and let
-me know!
-
-You should register for the conference at
-https://us.pycon.org/2018/registration/
+deadline. You can see that version of your talk here, {{url}}. Visiting this
+link will confirm that you've received this email and is critical for us to
+know you know you’ve been accepted.  Please review your proposal, and if you
+need to make changes to your title or abstract, please contact the team by
+emailing us at pycon-program-committee@python.org. We'll be announcing the
+list of accepted talks shortly. If you can no longer give this talk, please
+contact us by Feb 12th so that we can find a replacement.
+
+We haven't yet completed the schedule, so we don't yet know whether you've
+been allocated a 30 or a 45 minute time slot. We'll be announcing the schedule
+shortly. If you have particular scheduling requirements (if you plan to come
+to PyCon late or leave early, for example), please let us know by Feb 12th,
+so that we can take your requirements into account.
+
+You should register for the conference at https://us.pycon.org/2018/registration/
 as soon as possible; please don't delay, tickets do sell out!
 
-Unlike many conferences, PyCon doesn't give free tickets to speakers,
-nor do we cover travel expenses. We do this because PyCon is a
-grassroots, community-organized conference and our core mission is to
-keep costs low. Thus, we ask everybody who can afford to pay to do so.
-However, we recognize that buying tickets and paying for travel can
-represent a financial hardship to some, and we really want to see you
-at PyCon! If you can't afford a ticket and/or travel expenses,
-please apply for financial aid.
-
-More information about financial aid can be found at
-https://us.pycon.org/2018/financial-assistance/
-
-If you're applying for financial aid, make sure to mention that you're
-an accepted speaker.  Please contact the financial aid team at
-pycon-aid@python.org if you have any questions.
+PyCon provides speaker financial assistance to anyone that requests it. If you
+would like to get speaker financial assistance, please select the box on your
```

Comment:
I believe we also need to ask them to fill out a finaid app so we know what assistance they require.

Reply:
is that the official answer, because Ewa had commented on the google docs about using the checkbox on speaker profile?

Reply:
Started an email chain off of github to work towards the answer.